sections/dashboard/Overview/Overview.tsx

```diff
@@ -39,9 +48,117 @@ const Overview: FC = () => {
 	);
 	const [activeMarketsTab, setActiveMarketsTab] = useState<MarketsTab>(MarketsTab.FUTURES);
 
+	const { network, walletAddress, synthsMap } = Connector.useContainer();
+
+	const { tokenBalances } = useAppSelector(({ balances, exchange }) => ({
+		balancesMap: balances.balancesMap,
+		tokenList: exchange.tokenList,
+		tokenBalances: balances.tokenBalances,
+		balancesStatus: balances.status,
+	}));
```

Comment:
I think we can change this to:

```suggestion
	const tokenBalances = useAppSelector(({ balances }) => balances.tokenBalances);
```